### DISCUSSION TITLE
Is sitemap generator a bad approach for next.js?

### DISCUSSION THREAD

**Discussion by vasconx** [vercel/next.js#16935] [Help]
Hi, I'm new to this SEO web world! So I'm a little doubtful. I need to create a sitemap.xml for my website that contains some pages that will be updated daily, but there are few pages, nothing over 50. I searched on google and there are some websites that generate the sitemap. I did a test and got an xml file with all the paths that actually exist on my website. Is there anything bad about this approach that I'm not considering? Because I saw many issues talking about making the sitemap by code using streams, packages, etc., etc. And I was in doubt if just generating the sitemap on a site I found on google is a flawed solution for the next.js.

**Comment by diegoleft** [comment]
I'm currently using [nextjs-sitemap-generator](https://github.com/IlusionDev/nextjs-sitemap-generator) and is working perfectly well, take a look

**Reply by dazulu** [reply]
@diegoleft do you know if **nextjs-sitemap-generator** will ignore pages that have `noindex,nofollow` in them?
